snuba/datasets/entities/events.py

```diff
@@ -29,6 +32,11 @@
 # storage. Now we do not have entities so it is between dataset and
 # storage.
 event_translator = TranslationMappers(
+    columns=[
+        ColumnToMapping(None, "release", None, "tags", "sentry:release"),
+        ColumnToMapping(None, "dist", None, "tags", "sentry:dist"),
+        ColumnToMapping(None, "user", None, "tags", "sentry:user"),
+    ],
```

Comment:
Doesn't this mean that now the events entity must have these columns in the abstract schema ? Which it does not since it picks up the schema from the storage `storage.get_table_writer().get_schema()`

Reply:
Forgot about that, thanks